### ISSUE TITLE
`berkeley`: Update ETL code to handle projects with multiple sequencing runs

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc-runtime#647]
Berkeley schema now supports multiple sequencing runs via parthood on DataGeneration records. ETL code which generates records for subclasses of DataGeneration need to be updated to model this.

Example of new modeling, `nmdc:dgns-22-444xxx` is the parent record with two children.
```
{
    "data_generation_set": [
{
  "id": "nmdc:dgns-99-zUCd5N",
  "type": "nmdc:NucleotideSequencing",
  "analyte_category": "metagenome",
  "alternative_identifiers": [
    "gold:Gp0108335"
  ],
  "name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "has_input": [
    "nmdc:bsm-00-red"
  ],
  "has_output": [
    "nmdc:dobj-00-9n9n9n"
  ],
  "ncbi_project_name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "processing_institution": "JGI",
  "associated_studies": [
    "nmdc:sty-00-555xxx"
  ],
  "part_of": [
    "nmdc:dgns-22-444xxx"
  ]
},
  {"id": "nmdc:dgns-99-zUCd5Z",
  "type": "nmdc:NucleotideSequencing",
  "analyte_category": "metagenome",
  "alternative_identifiers": [
    "gold:Gp0108335"
  ],
  "name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "has_input": [
    "nmdc:bsm-00-red"
  ],
  "has_output": [
    "nmdc:dobj-00-9n9n9z"
  ],
  "ncbi_project_name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "processing_institution": "JGI",
  "associated_studies": [
    "nmdc:sty-00-555xxx"
  ],
  "part_of": [
    "nmdc:dgns-22-444xxx"
  ]
},
  {"id": "nmdc:dgns-22-444xxx",
  "type": "nmdc:NucleotideSequencing",
  "analyte_category": "metagenome",
  "alternative_identifiers": [
    "gold:Gp0108335"
  ],
  "name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "has_input": [
    "nmdc:bsm-00-red"
  ],
  "ncbi_project_name": "Thawing permafrost microbial communities from the Arctic, studying carbon transformations - Permafrost 712P3D",
  "processing_institution": "JGI",
  "associated_studies": [
    "nmdc:sty-00-555xxx"
  ]
}
]
}
```

**Comment by mslarae13** [comment]
@sujaypatil96 I know you're working on the ETL scripts this sprint.
When do you think you can start & finish this task?

@aclum remind me, this was a need for berkeley-roll out right? Is this MVP?

**Comment by aclum** [comment]
This is not part of the MVP and may get canceled depending on the metadata meeting tomorrow. Lower priority than getting gold_translator.py working.

**Comment by aclum** [comment]
blocked on decision about keeping or rolling back modeling.

**Comment by aclum** [comment]
NEON ETL was updated to use in_manifest on the DataObject records which is the current modeling in nmdc-schema version 11.1.*. GOLD ETL code needs to be modified but to get the information larger code changes are needed to look at both GOLD and JDP, not just code. Those changes are requested in https://github.com/microbiomedata/issues/issues/132
